demo/src/player/player-dialog.js

```diff
@@ -5,11 +5,15 @@
  * @module
  */
 import Pillarbox from '../../../src/pillarbox';
+import { createPlayer } from './player';
 
 const dialog = document.getElementById('pbw-dialog');
 
 // Pauses de video once the modal is closed.
-dialog.addEventListener('close', () => Pillarbox.getPlayer('player').pause());
+dialog.addEventListener('close', () => {
+  Pillarbox.getPlayer('player').dispose();
+  window.player = null;
```

Comment:
I would have created a `destroyPlayer` function and put it in the `player.js` file, but if you prefer it this way, that's fine too.

Reply:
I've integrated this suggestion too.